mods/voidmulti.sc2mod/base.sc2data/gamedata/actordata.xml

```diff
@@ -1837,7 +1861,7 @@
     </CActorSound>
     <CActorSimple id="RavenShredderMissileImpactArmorReductionTint">
         <On Terms="Behavior.RavenShredderMissileArmorReductionUISubtruct.On" Send="Create"/>
-        <On Terms="Behavior.RavenShredderMissileArmorReductionUISubtruct.On" Target="_Unit" Send="SetTintColor {255,100,0 2.000000} 0.100000 OneShot SeekerTargetArmor"/>
+        <On Terms="Behavior.RavenShredderMissileArmorReductionUISubtruct.On" Target="_Unit" Send="SetTintColor {255,187,85 2.000000} 0.100000 OneShot SeekerTargetArmor"/>
```

Comment:
**undocumented**: Changes the  the color from units affected by Raven's Anti-Armor-Missile from 255,100,0 to 255,187,85

old color: ![image](https://user-images.githubusercontent.com/5763784/206855164-bb444b44-a731-4d05-a82a-0905256b17c8.png) new color: ![image](https://user-images.githubusercontent.com/5763784/206855248-fc9d6ef1-95b9-4923-8b51-858ebf1ead88.png)